lib/modules/hosts/twitchclips.js

```diff
@@ -0,0 +1,16 @@
+export default {
+	moduleID: 'twitchclips',
+	name: 'twitch.tv clips',
+	domains: ['clips.twitch.tv'],
+	logo: 'https://www.twitch.tv/favicon.ico',
+	detect: ({ pathname }) => ((/^\/(\w+\/\w+)(?:\/|$)/).exec(pathname)),
```

Comment:
You can remove the extra parens on the outside of `((/^\/(\w+\/\w+)(?:\/|$)/).exec(pathname))`